README.md

```diff
@@ -0,0 +1,224 @@
+<div align="center" id='readme-top'>
```

Comment:
- [ ] Congratulations, all your frontend tests are passing:

![image](https://github.com/DavidEspinoG/mc_videogames_front/assets/57421823/b5a0810d-5884-4c88-9e1d-ab64aed641aa)

Now, kindly add a "Tests" section on your readme with instructions about how to run them properly. 👍